classes/local/store/s3/client.php

```diff
@@ -604,7 +604,7 @@ private function get_nice_filename($headers) {
 
             if (!empty($originalfilename)) {
                 $result['Content-Disposition'] = $contentdisposition;
-                $result['filename'] = "filename*=utf-8''" . rawurlencode($originalfilename);
+                $result['filename'] = 'filename="' . utf8_encode($originalfilename) . '"';
```

Comment:
no version micro bump?

Reply:
Hi @brendanheywood 

I have bump the version, same version as 33 Stable.
This branch will be replaced by 33 Stable so I guess it is not the issue